docs/canvas/canvas-common-functions.asciidoc

```diff
@@ -6,640 +6,765 @@ The common functions can run anywhere, which means they'll execute wherever
 
-A *** denotes a required argument.
+A *** denotes a required argument. 
 
+A *†* denotes an argument can be passed multiple times.
 
 [float]
-=== all
+[[all_fn]]
+=== `all`
 
-Returns true if all of the conditions are true.
+Returns `true` if all of the conditions are `true`. See also <<any_fn>>.
 
-*Accepts:* Anything or nothing
+*Accepts:* `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_*
-|boolean
+|_Unnamed_ *** *†*
+
+Alias: `condition`
+|`boolean`
 |The conditions to check
 |===
 
-*Returns:* boolean
+*Returns:* `boolean`
 
 
 [float]
-=== alterColumn
+[[alterColumn_fn]]
+=== `alterColumn`
 
-Converts between core types, for example, string, number, null, boolean, date 
-and rename columns.
+Converts between core types, for example, `string`, `number`, `null`, `boolean`, `date` and renames columns. See also <<mapColumn_fn>> and <<staticColumn_fn>>.
 
-*Accepts:* datatable
+*Accepts:* `datatable`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|column*
-|string
+|_Unnamed_ *** 
+
+Alias: `column`
+|`string`
 |The name of the column to alter
 
-|type
-|string
-|The type to convert the column to; leave blank to not change type (default: `null`)
+|`type`
+|`string`
+|The type to convert the column to; leave blank to not change type
 
-|name
-|string
-|The resultant column name; leave blank to not rename (default: `null`)
+|`name`
+|`string`
+|The resultant column name; leave blank to not rename
 |===
 
-*Returns:* datatable
+*Returns:* `datatable`
 
 
 [float]
-=== any
+[[any_fn]]
+=== `any`
 
-Returns true if any of the conditions are true.
+Returns `true` if any of the conditions are `true`. See also <<all_fn>>.
 
-*Accepts:* Anything or nothing
+*Accepts:* `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_*
-|boolean
+|_Unnamed_ *** *†*
+
+Alias: `condition`
+|`boolean`
 |The conditions to check
 |===
 
-*Returns:* boolean
+*Returns:* `boolean`
 
 
 [float]
-=== as
+[[as_fn]]
+=== `as`
 
-Creates a datatable with a single value.
+Creates a `datatable` with a single value. See also <<getCell_fn>>.
 
-*Accepts:* string, boolean, number, null
+*Accepts:* `string`, `boolean`, `number`, `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_ (Alias: `name`)
-|string
-|A name to give the column (default: `value`)
+|_Unnamed_ 
+
+Alias: `name`
+|`string`
+|A name to give the column 
+
+Default: `"value"`
 |===
 
-*Returns:* datatable
+*Returns:* `datatable`
 
 
 [float]
-=== axisConfig
+[[axisConfig_fn]]
+=== `axisConfig`
 
-Configures axis of a visualization.
+Configures axis of a visualization, only used in <<plot_fn>>
 
-*Accepts:* datatable
+*Accepts:* `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|show
-|boolean
-|default: `true`
+|`show`
+|`boolean`
+|Show the axis labels? 
+
+Default: `true`
+
+|`position`
+|`string`
+|The position of the axis labels, for example, top, bottom, left, and right 
```

Comment:
Could we split this into two sentences. For example:

`The position of the axis labels. For example, top, bottom, left, or right.`